README.md

```diff
@@ -52,6 +52,9 @@ Kubernetes 1.14
  * `--allow-privileged` flag must be set to true for the API server
  * `--feature-gates=ExpandCSIVolumes=true,ExpandInUsePersistentVolumes=true` feature gate flags must be set to true for both the API server and kubelet
 
+Kubernetes 1.15
+ * `--feature-gates=CSIInlineVolume=true,VolumePVCDataSource=true` feature gate flags must be set to true for both the API server and kubelet
```

Comment:
Add " for ephemeral inline volume and Volume cloning" . Or separate them into 2 different bullet points.